homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -77,7 +77,7 @@ async def async_step_confirm(
             (manufacturer, device_type, model) = await Mower(
                 channel_id, self.address
             ).probe_gatts(device)
-        except (BleakError, TimeoutError) as exception:
+        except (BleakDBusError, BleakError, TimeoutError) as exception:
```

Comment:
`BleakDBusError` extends from  `BleakError`. Why are both needed here?

Reply:
I'm not sure, it just seems like that's the error that people are seeing here: https://github.com/home-assistant/core/issues/143827#issuecomment-2838672170

Reply:
and both are documented by `bleak connect()`: https://bleak.readthedocs.io/en/latest/backends/linux.html#bleak.backends.bluezdbus.client.BleakClientBlueZDBus.connect

Reply:
That exception is already being catch here. You can see that it has the `Failed to connect to device: ...` log.

Reply:
It is being logged with logger.exception, which is why its showing up with backtrace